### DISCUSSION TITLE
DeDRM - PDF Conversion Failed

### DISCUSSION THREAD

**Discussion by calliopejane** [apprenticeharper/DeDRM_tools#1538] [Q&A]
Trying to convert a pdf with DRM in Calibre 5.11.0.  
The process fails with the details below. Help?

Convert book 1 of 1 (Klan-destine relationships a bl)
DeDRM v7.0.3: Trying to decrypt _p3ahs9c.pdf
DeDRM v7.0.3: _p3ahs9c.pdf is a PDF ebook
DeDRM v7.0.3: Looking for new default Adobe Digital Editions Keys after 0.0 seconds
Found 1 keys
DeDRM v7.0.3: Trying a new default key
ebx_V is 4  and ebx_type is 6
length is 16 and len(bookkey) is 14
bookkey[0] is 158
DeDRM v7.0.3: Exception when decrypting after 0.1 seconds
Traceback (most recent call last):
  File "calibre_plugins.dedrm.__init__", line 484, in PDFDecrypt
  File "calibre_plugins.dedrm.ineptpdf", line 2179, in decryptBook
  File "calibre_plugins.dedrm.ineptpdf", line 1985, in __init__
  File "calibre_plugins.dedrm.ineptpdf", line 1469, in initialize
  File "calibre_plugins.dedrm.ineptpdf", line 1623, in initialize_ebx
calibre_plugins.dedrm.ineptpdf.ADEPTError: error decrypting book session key - mismatched length
DeDRM v7.0.3: Failed to decrypt with new default key after 0.1 seconds
DeDRM v7.0.3: Ultimately failed to decrypt after 0.1 seconds. Read the FAQs at Harper's repository: https://github.com/apprenticeharper/DeDRM_tools/blob/master/FAQs.md
Conversion options changed from defaults:
  read_metadata_from_opf: 'C:\\Users\\yonib\\AppData\\Local\\Temp\\calibre_hkk9le5n\\f_f3rjtr.opf'
  verbose: 2
Resolved conversion options
calibre version: 5.11.0
{'asciiize': False,
 'author_sort': None,
 'authors': None,
 'base_font_size': 0.0,
 'book_producer': None,
 'change_justification': 'original',
 'chapter': "//*[((name()='h1' or name()='h2') and re:test(., "
            "'\\s*((chapter|book|section|part)\\s+)|((prolog|prologue|epilogue)(\\s+|$))', "
            "'i')) or @class = 'chapter']",
 'chapter_mark': 'pagebreak',
 'comments': None,
 'cover': None,
 'debug_pipeline': None,
 'dehyphenate': True,
 'delete_blank_paragraphs': True,
 'disable_font_rescaling': False,
 'dont_compress': False,
 'duplicate_links_in_toc': False,
 'embed_all_fonts': False,
 'embed_font_family': None,
 'enable_heuristics': False,
 'expand_css': False,
 'extra_css': None,
 'extract_to': None,
 'filter_css': '',
 'fix_indents': True,
 'font_size_mapping': None,
 'format_scene_breaks': True,
 'html_unwrap_factor': 0.4,
 'input_encoding': None,
 'input_profile': <calibre.customize.profiles.InputProfile object at 0x08545628>,
 'insert_blank_line': False,
 'insert_blank_line_size': 0.5,
 'insert_metadata': False,
 'isbn': None,
 'italicize_common_cases': True,
 'keep_ligatures': False,
 'language': None,
 'level1_toc': None,
 'level2_toc': None,
 'level3_toc': None,
 'line_height': 0.0,
 'linearize_tables': False,
 'margin_bottom': 5.0,
 'margin_left': 5.0,
 'margin_right': 5.0,
 'margin_top': 5.0,
 'markup_chapter_headings': True,
 'max_toc_links': 50,
 'minimum_line_height': 120.0,
 'mobi_toc_at_start': False,
 'new_pdf_engine': False,
 'no_chapters_in_toc': False,
 'no_images': False,
 'no_inline_navbars': False,
 'no_inline_toc': False,
 'output_profile': <calibre.customize.profiles.OutputProfile object at 0x08545790>,
 'page_breaks_before': "//*[name()='h1' or name()='h2']",
 'prefer_author_sort': False,
 'prefer_metadata_cover': False,
 'pretty_print': False,
 'pubdate': None,
 'publisher': None,
 'rating': None,
 'read_metadata_from_opf': 'C:\\Users\\yonib\\AppData\\Local\\Temp\\calibre_hkk9le5n\\f_f3rjtr.opf',
 'remove_fake_margins': True,
 'remove_first_image': False,
 'remove_paragraph_spacing': False,
 'remove_paragraph_spacing_indent_size': 1.5,
 'renumber_headings': True,
 'replace_scene_breaks': '',
 'search_replace': '[]',
 'series': None,
 'series_index': None,
 'share_not_sync': False,
 'smarten_punctuation': False,
 'sr1_replace': None,
 'sr1_search': None,
 'sr2_replace': None,
 'sr2_search': None,
 'sr3_replace': None,
 'sr3_search': None,
 'start_reading_at': None,
 'subset_embedded_fonts': False,
 'tags': None,
 'timestamp': None,
 'title': None,
 'title_sort': None,
 'toc_filter': None,
 'toc_threshold': 6,
 'toc_title': None,
 'transform_css_rules': '[]',
 'unsmarten_punctuation': False,
 'unwrap_factor': 0.45,
 'unwrap_lines': True,
 'use_auto_toc': False,
 'verbose': 2}
Running file type plugin DeDRM failed with traceback:
Traceback (most recent call last):
  File "calibre\customize\ui.py", line 172, in _run_filetype_plugins
  File "calibre_plugins.dedrm.__init__", line 637, in run
  File "calibre_plugins.dedrm.__init__", line 512, in PDFDecrypt
calibre_plugins.dedrm.DeDRMError: DeDRM v7.0.3: Ultimately failed to decrypt after 0.1 seconds. Read the FAQs at Harper's repository: https://github.com/apprenticeharper/DeDRM_tools/blob/master/FAQs.md
DeDRM v7.0.3: Trying to decrypt _p3ahs9c.pdf
DeDRM v7.0.3: _p3ahs9c.pdf is a PDF ebook
DeDRM v7.0.3: Looking for new default Adobe Digital Editions Keys after 0.0 seconds
Found 1 keys
DeDRM v7.0.3: Trying a new default key
ebx_V is 4  and ebx_type is 6
length is 16 and len(bookkey) is 14
bookkey[0] is 158
DeDRM v7.0.3: Exception when decrypting after 0.1 seconds
Traceback (most recent call last):
  File "calibre_plugins.dedrm.__init__", line 484, in PDFDecrypt
  File "calibre_plugins.dedrm.ineptpdf", line 2179, in decryptBook
  File "calibre_plugins.dedrm.ineptpdf", line 1985, in __init__
  File "calibre_plugins.dedrm.ineptpdf", line 1469, in initialize
  File "calibre_plugins.dedrm.ineptpdf", line 1623, in initialize_ebx
calibre_plugins.dedrm.ineptpdf.ADEPTError: error decrypting book session key - mismatched length
DeDRM v7.0.3: Failed to decrypt with new default key after 0.1 seconds
DeDRM v7.0.3: Ultimately failed to decrypt after 0.1 seconds. Read the FAQs at Harper's repository: https://github.com/apprenticeharper/DeDRM_tools/blob/master/FAQs.md
Running file type plugin DeDRM failed with traceback:
Traceback (most recent call last):
  File "calibre\customize\ui.py", line 172, in _run_filetype_plugins
  File "calibre_plugins.dedrm.__init__", line 637, in run
  File "calibre_plugins.dedrm.__init__", line 512, in PDFDecrypt
calibre_plugins.dedrm.DeDRMError: DeDRM v7.0.3: Ultimately failed to decrypt after 0.1 seconds. Read the FAQs at Harper's repository: https://github.com/apprenticeharper/DeDRM_tools/blob/master/FAQs.md
InputFormatPlugin: PDF Input running
on C:\Users\yonib\AppData\Local\Temp\calibre_hkk9le5n\_p3ahs9c.pdf
Converting file to html...
Traceback (most recent call last):
  File "runpy.py", line 194, in _run_module_as_main
  File "runpy.py", line 87, in _run_code
  File "site.py", line 82, in <module>
  File "site.py", line 77, in main
  File "site.py", line 49, in run_entry_point
  File "calibre\utils\ipc\worker.py", line 216, in main
  File "calibre\gui2\convert\gui_conversion.py", line 41, in gui_convert_override
  File "calibre\gui2\convert\gui_conversion.py", line 28, in gui_convert
  File "calibre\ebooks\conversion\plumber.py", line 1108, in run
  File "calibre\customize\conversion.py", line 245, in __call__
  File "calibre\ebooks\conversion\plugins\pdf_input.py", line 54, in convert
  File "calibre\ebooks\pdf\pdftohtml.py", line 83, in pdftohtml
calibre.ebooks.ConversionError: pdftohtml failed with return code: 1
Syntax Error: Couldn't find the 'EBX_HANDLER' security handler
Exception ignored in: <function NativeFunction.__del__ at 0x08763850>
Traceback (most recent call last):
  File "calibre_plugins.dedrm.adobekey", line 266, in __del__
TypeError: 'NoneType' object is not callable
Exception ignored in: <function NativeFunction.__del__ at 0x08763850>
Traceback (most recent call last):
  File "calibre_plugins.dedrm.adobekey", line 266, in __del__
TypeError: 'NoneType' object is not callable

**Comment by ElleKayEm** [comment]
Have you tried with calibre 4.23 and DeDRM 6.8.1?

**Comment by Rider71-del** [comment]
I am also trying to convert a PDF with DRM in Calibre 5.13.0.
The process takes a really long time is there an error? 

Buch 1 von 1 (Batman - Der Rat der Eulen) konvertieren
DeDRM v7.1.0: Trying to decrypt j5sasstl.pdf
DeDRM v7.1.0: j5sasstl.pdf is a PDF ebook
DeDRM v7.1.0: Trying Encryption key default_key
DeDRM v7.1.0: Exception when decrypting after 0.0 seconds
Traceback (most recent call last):
  File "calibre_plugins.dedrm.__init__", line 434, in PDFDecrypt
  File "calibre_plugins.dedrm.ineptpdf", line 2178, in decryptBook
  File "calibre_plugins.dedrm.ineptpdf", line 1984, in __init__
  File "calibre_plugins.dedrm.ineptpdf", line 1467, in initialize
  File "calibre_plugins.dedrm.ineptpdf", line 1569, in initialize_standard
calibre_plugins.dedrm.ineptpdf.ADEPTError: Password is not correct.
DeDRM v7.1.0: Failed to decrypt with key default_key after 0.0 seconds
DeDRM v7.1.0: Trying Encryption key default_key_2
DeDRM v7.1.0: Exception when decrypting after 0.0 seconds
Traceback (most recent call last):
  File "calibre_plugins.dedrm.__init__", line 434, in PDFDecrypt
  File "calibre_plugins.dedrm.ineptpdf", line 2178, in decryptBook
  File "calibre_plugins.dedrm.ineptpdf", line 1984, in __init__
  File "calibre_plugins.dedrm.ineptpdf", line 1467, in initialize
  File "calibre_plugins.dedrm.ineptpdf", line 1569, in initialize_standard
calibre_plugins.dedrm.ineptpdf.ADEPTError: Password is not correct.
DeDRM v7.1.0: Failed to decrypt with key default_key_2 after 0.1 seconds
DeDRM v7.1.0: Looking for new default Adobe Digital Editions Keys after 0.1 seconds
Found 1 keys
DeDRM v7.1.0: Trying a new default key
DeDRM v7.1.0: Exception when decrypting after 0.1 seconds
Traceback (most recent call last):
  File "calibre_plugins.dedrm.__init__", line 485, in PDFDecrypt
  File "calibre_plugins.dedrm.ineptpdf", line 2178, in decryptBook
  File "calibre_plugins.dedrm.ineptpdf", line 1984, in __init__
  File "calibre_plugins.dedrm.ineptpdf", line 1467, in initialize
  File "calibre_plugins.dedrm.ineptpdf", line 1569, in initialize_standard
calibre_plugins.dedrm.ineptpdf.ADEPTError: Password is not correct.
DeDRM v7.1.0: Failed to decrypt with new default key after 0.1 seconds
DeDRM v7.1.0: Ultimately failed to decrypt after 0.1 seconds. Read the FAQs at Harper's repository: https://github.com/apprenticeharper/DeDRM_tools/blob/master/FAQs.md
Conversion options changed from defaults:
  cover: 'C:\\Users\\Hellb\\AppData\\Local\\Temp\\calibre_7cdr28c9\\olozp9om.jpeg'
  verbose: 2
  read_metadata_from_opf: 'C:\\Users\\Hellb\\AppData\\Local\\Temp\\calibre_7cdr28c9\\2sioa62z.opf'
  output_profile: 'cybook_opus'
Resolved conversion options
calibre version: 5.13.0
{'asciiize': False,
 'author_sort': None,
 'authors': None,
 'base_font_size': 0.0,
 'book_producer': None,
 'change_justification': 'original',
 'chapter': "//*[((name()='h1' or name()='h2') and re:test(., "
            "'\\s*((chapter|book|section|part)\\s+)|((prolog|prologue|epilogue)(\\s+|$))', "
            "'i')) or @class = 'chapter']",
 'chapter_mark': 'pagebreak',
 'comments': None,
 'cover': 'C:\\Users\\Hellb\\AppData\\Local\\Temp\\calibre_7cdr28c9\\olozp9om.jpeg',
 'debug_pipeline': None,
 'dehyphenate': True,
 'delete_blank_paragraphs': True,
 'disable_font_rescaling': False,
 'duplicate_links_in_toc': False,
 'embed_all_fonts': False,
 'embed_font_family': None,
 'enable_heuristics': False,
 'expand_css': False,
 'extra_css': None,
 'extract_to': None,
 'filter_css': '',
 'fix_indents': True,
 'font_size_mapping': None,
 'format_scene_breaks': True,
 'html_unwrap_factor': 0.4,
 'input_encoding': None,
 'input_profile': <calibre.customize.profiles.InputProfile object at 0x000001B885608400>,
 'insert_blank_line': False,
 'insert_blank_line_size': 0.5,
 'insert_metadata': False,
 'isbn': None,
 'italicize_common_cases': True,
 'keep_ligatures': False,
 'language': None,
 'level1_toc': None,
 'level2_toc': None,
 'level3_toc': None,
 'line_height': 0.0,
 'linearize_tables': False,
 'margin_bottom': 5.0,
 'margin_left': 5.0,
 'margin_right': 5.0,
 'margin_top': 5.0,
 'markup_chapter_headings': True,
 'max_toc_links': 50,
 'minimum_line_height': 120.0,
 'new_pdf_engine': False,
 'no_chapters_in_toc': False,
 'no_images': False,
 'no_inline_navbars': False,
 'output_profile': <calibre.customize.profiles.CybookOpusOutput object at 0x000001B8856086A0>,
 'page_breaks_before': "//*[name()='h1' or name()='h2']",
 'prefer_metadata_cover': False,
 'pretty_print': True,
 'pubdate': None,
 'publisher': None,
 'rating': None,
 'read_metadata_from_opf': 'C:\\Users\\Hellb\\AppData\\Local\\Temp\\calibre_7cdr28c9\\2sioa62z.opf',
 'remove_fake_margins': True,
 'remove_first_image': False,
 'remove_paragraph_spacing': False,
 'remove_paragraph_spacing_indent_size': 1.5,
 'renumber_headings': True,
 'replace_scene_breaks': '',
 'search_replace': '[]',
 'series': None,
 'series_index': None,
 'smarten_punctuation': False,
 'sr1_replace': None,
 'sr1_search': None,
 'sr2_replace': None,
 'sr2_search': None,
 'sr3_replace': None,
 'sr3_search': None,
 'start_reading_at': None,
 'subset_embedded_fonts': False,
 'tags': None,
 'template_css': None,
 'template_html': None,
 'template_html_index': None,
 'timestamp': None,
 'title': None,
 'title_sort': None,
 'toc_filter': None,
 'toc_threshold': 6,
 'transform_css_rules': '[]',
 'unsmarten_punctuation': False,
 'unwrap_factor': 0.45,
 'unwrap_lines': True,
 'use_auto_toc': False,
 'verbose': 2}
Running file type plugin DeDRM failed with traceback:
Traceback (most recent call last):
  File "calibre\customize\ui.py", line 172, in _run_filetype_plugins
  File "calibre_plugins.dedrm.__init__", line 638, in run
  File "calibre_plugins.dedrm.__init__", line 513, in PDFDecrypt
calibre_plugins.dedrm.DeDRMError: DeDRM v7.1.0: Ultimately failed to decrypt after 0.1 seconds. Read the FAQs at Harper's repository: https://github.com/apprenticeharper/DeDRM_tools/blob/master/FAQs.md
DeDRM v7.1.0: Trying to decrypt j5sasstl.pdf
DeDRM v7.1.0: j5sasstl.pdf is a PDF ebook
DeDRM v7.1.0: Trying Encryption key default_key
DeDRM v7.1.0: Exception when decrypting after 0.0 seconds
Traceback (most recent call last):
  File "calibre_plugins.dedrm.__init__", line 434, in PDFDecrypt
  File "calibre_plugins.dedrm.ineptpdf", line 2178, in decryptBook
  File "calibre_plugins.dedrm.ineptpdf", line 1984, in __init__
  File "calibre_plugins.dedrm.ineptpdf", line 1467, in initialize
  File "calibre_plugins.dedrm.ineptpdf", line 1569, in initialize_standard
calibre_plugins.dedrm.ineptpdf.ADEPTError: Password is not correct.
DeDRM v7.1.0: Failed to decrypt with key default_key after 0.0 seconds
DeDRM v7.1.0: Trying Encryption key default_key_2
DeDRM v7.1.0: Exception when decrypting after 0.0 seconds
Traceback (most recent call last):
  File "calibre_plugins.dedrm.__init__", line 434, in PDFDecrypt
  File "calibre_plugins.dedrm.ineptpdf", line 2178, in decryptBook
  File "calibre_plugins.dedrm.ineptpdf", line 1984, in __init__
  File "calibre_plugins.dedrm.ineptpdf", line 1467, in initialize
  File "calibre_plugins.dedrm.ineptpdf", line 1569, in initialize_standard
calibre_plugins.dedrm.ineptpdf.ADEPTError: Password is not correct.
DeDRM v7.1.0: Failed to decrypt with key default_key_2 after 0.0 seconds
DeDRM v7.1.0: Looking for new default Adobe Digital Editions Keys after 0.0 seconds
Found 1 keys
DeDRM v7.1.0: Trying a new default key
DeDRM v7.1.0: Exception when decrypting after 0.0 seconds
Traceback (most recent call last):
  File "calibre_plugins.dedrm.__init__", line 485, in PDFDecrypt
  File "calibre_plugins.dedrm.ineptpdf", line 2178, in decryptBook
  File "calibre_plugins.dedrm.ineptpdf", line 1984, in __init__
  File "calibre_plugins.dedrm.ineptpdf", line 1467, in initialize
  File "calibre_plugins.dedrm.ineptpdf", line 1569, in initialize_standard
calibre_plugins.dedrm.ineptpdf.ADEPTError: Password is not correct.
DeDRM v7.1.0: Failed to decrypt with new default key after 0.0 seconds
DeDRM v7.1.0: Ultimately failed to decrypt after 0.0 seconds. Read the FAQs at Harper's repository: https://github.com/apprenticeharper/DeDRM_tools/blob/master/FAQs.md
Running file type plugin DeDRM failed with traceback:
Traceback (most recent call last):
  File "calibre\customize\ui.py", line 172, in _run_filetype_plugins
  File "calibre_plugins.dedrm.__init__", line 638, in run
  File "calibre_plugins.dedrm.__init__", line 513, in PDFDecrypt
calibre_plugins.dedrm.DeDRMError: DeDRM v7.1.0: Ultimately failed to decrypt after 0.0 seconds. Read the FAQs at Harper's repository: https://github.com/apprenticeharper/DeDRM_tools/blob/master/FAQs.md
InputFormatPlugin: PDF Input running
on C:\Users\Hellb\AppData\Local\Temp\calibre_7cdr28c9\j5sasstl.pdf
Converting file to html...

**Reply by ElleKayEm** [reply]
Adobe Digital Editions is authorized with your Adobe ID and on the same machine and user as calibre with DeDRM?

**Comment by Rider71-del** [comment]
yes is why ?

**Reply by ElleKayEm** [reply]
Have you tried with calibre 4.23 and DeDRM 6.8.1?

**Comment by Rider71-del** [comment]
No, I haven't yet, but I'll give it a try 
do you still get the versions of calibre 4.23 and DeDRM 6.8.1?

**Reply by Rider71-del** [reply]
ok found

**Comment by Rider71-del** [comment]
so hatt not brought he now conventerit the pdf in ebup 12 minuten scon and it tuts nothing 
her the log


Buch 1 von 1 (Batman - Der Rat der Eulen) konvertieren
C:\Program Files\Calibre2\\app\pylib.zip\dateutil\parser\_parser.py:1177: UnicodeWarning: Unicode equal comparison failed to convert both arguments to Unicode - interpreting them as being unequal
Conversion options changed from defaults:
  read_metadata_from_opf: u'C:\\Users\\Hellb\\AppData\\Local\\Temp\\calibre_vssx38\\paodwn.opf'
  output_profile: u'cybook_opus'
  cover: u'C:\\Users\\Hellb\\AppData\\Local\\Temp\\calibre_vssx38\\ipf79p.jpeg'
  verbose: 2
Resolved conversion options
calibre version: 4.23.0
{'asciiize': False,
 'author_sort': None,
 'authors': None,
 'base_font_size': 0.0,
 'book_producer': None,
 'change_justification': u'original',
 'chapter': u"//*[((name()='h1' or name()='h2') and re:test(., '\\s*((chapter|book|section|part)\\s+)|((prolog|prologue|epilogue)(\\s+|$))', 'i')) or @class = 'chapter']",
 'chapter_mark': u'pagebreak',
 'comments': None,
 'cover': u'C:\\Users\\Hellb\\AppData\\Local\\Temp\\calibre_vssx38\\ipf79p.jpeg',
 'debug_pipeline': None,
 'dehyphenate': True,
 'delete_blank_paragraphs': True,
 'disable_font_rescaling': False,
 'dont_split_on_page_breaks': False,
 'duplicate_links_in_toc': False,
 'embed_all_fonts': False,
 'embed_font_family': None,
 'enable_heuristics': False,
 'epub_flatten': False,
 'epub_inline_toc': False,
 'epub_toc_at_end': False,
 'epub_version': u'2',
 'expand_css': False,
 'extra_css': None,
 'extract_to': None,
 'filter_css': u'',
 'fix_indents': True,
 'flow_size': 260,
 'font_size_mapping': None,
 'format_scene_breaks': True,
 'html_unwrap_factor': 0.4,
 'input_encoding': None,
 'input_profile': <calibre.customize.profiles.InputProfile object at 0x000001A6C5C60E48>,
 'insert_blank_line': False,
 'insert_blank_line_size': 0.5,
 'insert_metadata': False,
 'isbn': None,
 'italicize_common_cases': True,
 'keep_ligatures': False,
 'language': None,
 'level1_toc': None,
 'level2_toc': None,
 'level3_toc': None,
 'line_height': 0.0,
 'linearize_tables': False,
 'margin_bottom': 5.0,
 'margin_left': 5.0,
 'margin_right': 5.0,
 'margin_top': 5.0,
 'markup_chapter_headings': True,
 'max_toc_links': 50,
 'minimum_line_height': 120.0,
 'new_pdf_engine': False,
 'no_chapters_in_toc': False,
 'no_default_epub_cover': False,
 'no_images': False,
 'no_inline_navbars': False,
 'no_svg_cover': False,
 'output_profile': <calibre.customize.profiles.CybookOpusOutput object at 0x000001A6C5C7D908>,
 'page_breaks_before': u"//*[name()='h1' or name()='h2']",
 'prefer_metadata_cover': False,
 'preserve_cover_aspect_ratio': False,
 'pretty_print': True,
 'pubdate': None,
 'publisher': None,
 'rating': None,
 'read_metadata_from_opf': u'C:\\Users\\Hellb\\AppData\\Local\\Temp\\calibre_vssx38\\paodwn.opf',
 'remove_fake_margins': True,
 'remove_first_image': False,
 'remove_paragraph_spacing': False,
 'remove_paragraph_spacing_indent_size': 1.5,
 'renumber_headings': True,
 'replace_scene_breaks': u'',
 'search_replace': '[]',
 'series': None,
 'series_index': None,
 'smarten_punctuation': False,
 'sr1_replace': None,
 'sr1_search': None,
 'sr2_replace': None,
 'sr2_search': None,
 'sr3_replace': None,
 'sr3_search': None,
 'start_reading_at': None,
 'subset_embedded_fonts': False,
 'tags': None,
 'timestamp': None,
 'title': None,
 'title_sort': None,
 'toc_filter': None,
 'toc_threshold': 6,
 'toc_title': None,
 'transform_css_rules': '[]',
 'unsmarten_punctuation': False,
 'unwrap_factor': 0.45,
 'unwrap_lines': True,
 'use_auto_toc': False,
 'verbose': 2}
InputFormatPlugin: PDF Input running
on C:\Users\Hellb\AppData\Local\Temp\calibre_vssx38\zccfuc.pdf
Converting file to html...

**Reply by ElleKayEm** [reply]
Let's see an import log:  https://github.com/apprenticeharper/DeDRM_tools/blob/master/FAQs.md#i-cannot-solve-my-problem-with-the-dedrm-plugin-and-now-i-need-to-post-a-log-how-do-i-do-that

**Reply by Rider71-del** [reply]
here the log 

[tmpshcfec.txt](https://github.com/apprenticeharper/DeDRM_tools/files/6162890/tmpshcfec.txt)

**Reply by ElleKayEm** [reply]
You're still getting this in the log:

> ADEPTError: Password is not correct.

I don't remember seeing anything like that before.  Is there some kind of extra security on this?  Did you use ADE 2.0.1 to open the ACSM file?

**Reply by Rider71-del** [reply]
I don't know if there is any extra security. First of all, I don't have an ACSM file. I have a PDF file that I can download from the Pocketbook site where I bought it. I open the eBook with ADE 2.0.1.

**Reply by Rider71-del** [reply]
Here is a picture 
As you can see, there are 2 ADE accounts, but Hellboy81 is set as the default. 
![Unbenannt](https://user-images.githubusercontent.com/80758500/111617951-159f8080-87e4-11eb-9cab-aaf8654717a7.PNG)

**Reply by ElleKayEm** [reply]
It seems like this book isn't tied to your Adobe ID.  I don't know how you can remove the DRM.

**Reply by calliopejane** [reply]
I think my problem with the book I couldn't convert might have been because it was a borrowed e-book from the Internet Archive's Open Library, so it had a library time limit on it as well as Adobe's DRM.  
I wasn't trying to permanently keep the book or make multiple copies; I just wanted to read it on my kindle.  But after a few more unsuccessful attempts to convert, I just gave up and read the book on an inconvenient device.

Also, not sure if this has anything to do with it, but the book was an old out-of-print thing, and the conversion to e-book seemed rather crude.  Even though the text was recognized as text and chapters were indexed, the pages *looked* more like a low-res image scan. 
Reading it was rough, but I was really interested in the content and this was the only way I found to read the book (the cheapest used physical copy I found was $600!)

**Reply by Rider71-del** [reply]
> It seems like this book isn't tied to your Adobe ID. I don't know how you can remove the DRM.
If you want I can send you the pdf file so you can check it yourself. 

maybe they will find out  ?

**Reply by ElleKayEm** [reply]
@Rider71-del
Pretty sure that's the problem.  You need to open an acsm file with ADE that's authorized with your Adobe ID.  Then the epub or pdf that is downloaded can be decrypted.

**Reply by Rider71-del** [reply]
when on the shop page https://pocketbook.de/de_en/batman-band-1-der-rat-der-eulen-9783736710313 you do not download an acsm file but the pdf file. 

I can send you the pdf if you tell me where to send it. ?

**Reply by ElleKayEm** [reply]
No, that won't work.  Like I said, this book isn't linked to your Adobe ID.  I do not know how you can remove the DRM from a book that is tied to a vendor account like this.
